### PR TITLE
Many improvements

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
Thank you for all your hard work on TST.

The readme needed much improvement, so I improved it!

For the most part, I only had sufficient time to improve above the "FAQ" section, but it's now much better above that section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,36 +4,35 @@
...
 If you want, you can restructure the tree via drag and drop.
 
  * Such a tree of tabs will behave like a visual browsing history for you.
-   For example, if you see a list of search results for a topic, you'll open each search result link in new child tab.
-   For more details you'll also open more descendant tabs from them.
+   For example, if you see a list of search results for a topic, each search result link will open in new child tab.
+   New tabs opened from these "child" tabs will appear as descendants of their originating tabs.
    You'll easily dig and dig deeply, without losing your browsing trail - if you want to go back to the original search result, you just have to switch to the "root" tab.
- * Moreover, you'll treat a tree of tabs just as "grouped tabs" for similar topics.
+ * Moreover, you'll each tree will reflect a group of tabs on a similar topic.
```
Comment (piroor) [inline README.md]: Missing "treat" ?

**Comment by piroor** [comment]
Thank you for improving. I have one confusion and commented it. Could you check that?

File: README.md (hunk 1 above)
Comment (Gitoffthelawn) [reply README.md]: Ah, thanks!  Actually, the "treat" should be removed, but I missed also removing the "you'll" right before it.  I'll fix it now for you.

**Review by Gitoffthelawn — COMMENTED** [review]
Fixed!

**Comment by piroor** [comment]
Thanks!

**Comment by Gitoffthelawn** [comment]
> Thanks!

You're most welcome!  And thank *you* for the amazing extension!

**Pull request merged**